<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>761</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>731</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="background" stdset="0">
     <string>transparent</string>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Редактирование</string>
     </attribute>
     <widget class="QPushButton" name="loadFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>480</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Загрузить
данные из файла</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveFile">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить
данные в файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>390</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Стоимость</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Район</string>
      </property>
     </widget>
     <widget class="QComboBox" name="type">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>91</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Студия</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1-комнатная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2-комнатная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3-комнатная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Более 3 комнат</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="district">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="cost">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Тип</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="is_bought">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Квартира куплена</string>
      </property>
      <widget class="QRadioButton" name="yes_bought">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Да</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="no_bought">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Нет</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ФИО клиента</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_name">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Тел. клиента</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_phone">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить информацию о квартире</string>
      </property>
     </widget>
     <widget class="QComboBox" name="adressList">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Просмотр</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Результаты поиска</string>
      </property>
     </widget>
     <widget class="QPushButton" name="find_dist">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>200</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти квартиру
по району</string>
      </property>
     </widget>
     <widget class="QPushButton" name="find">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>70</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти по
типу квартиры</string>
      </property>
     </widget>
     <widget class="QPushButton" name="getAllHouses">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>290</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Вывести список
всех квартир</string>
      </property>
     </widget>
     <widget class="QComboBox" name="type_find">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>30</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Студия</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1-комнатная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2-комнатная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3-комнатная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Более 3 комнат</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="district_find">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>160</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableOut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1041</width>
        <height>631</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>65</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>53</number>
      </attribute>
      <column>
       <property name="text">
        <string>Адрес</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Район</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Тип</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Стоимость</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Куплена</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ФИО клиента</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Телефон клиента</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="clearHouses">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>380</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить 
список квартир</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Адреса</string>
     </attribute>
     <widget class="QLineEdit" name="new_adress">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить адрес в базу данных</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_adress">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadFile_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Загрузить
данные из файла</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveFile_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить
данные в файл</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="out_adress">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>611</width>
        <height>521</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string>Адреса</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="show_adress">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Показать адреса</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Список адресов</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearAdresses">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить
список адресов</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
